ajax/libs/stretchy/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "stretchy",
+  "version": "1.1.0",
+  "description": "Form element autosizing, the way it should be.",
+  "filename": "stretchy.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LeaVerou/stretchy.git"
+  },
+  "author": "Lea Verou",
+  "license": "MIT",
+  "keywords": [
+    "element autosizing"
```

Comment:
Could you please enrich and modify `keywords` with `"form"`, `"element"`, `"autosizing"`. Thanks. (We hope there are at least 3 keywords in package.json)